vixenNetwork.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: "Vixen/Blacked/Tushy"
```

Comment:
```suggestion
name: "Vixen Media Group"
```
Instead of the name of this scraper ever expanding this is probably more inclusive as the official name. Can maybe add a comment to the top of the tile though to the extent of:
```
# This currently includes Blacked, BlackedRaw, Tushy, TushyRaw,
# Vixen, and Deeper.
```